### DISCUSSION TITLE
[Suggestion] Petio

### DISCUSSION THREAD

**Discussion by YukiRyuu** [tteck/Proxmox#3136] [Ideas]
I'd love to see a one liner script for Petio to complete the Media collection of Plex + Radarr + Sonarr etc. and allow users to request and autodownload Movies/Series

**Comment by nfribeiro** [comment]
I got an error while installing.

 ✓ Installed Dependencies
Hit:1 http://security.debian.org bookworm-security InRelease
Hit:2 http://deb.debian.org/debian bookworm InRelease    
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Ign:4 http://repo.mongodb.org/apt/debian bookworm/mongodb-org/4.4 InRelease
Get:5 http://repo.mongodb.org/apt/debian bookworm/mongodb-org/4.4 Release [1,991 B]
Get:6 http://repo.mongodb.org/apt/debian bookworm/mongodb-org/4.4 Release.gpg [866 B]
Get:7 http://repo.mongodb.org/apt/debian bookworm/mongodb-org/4.4/main amd64 Packages [15.2 kB]
Fetched 18.1 kB in 1s (13.5 kB/s)  
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package mongodb-org

[ERROR] in line 35: exit code 0: while executing command $STD apt-get install -y mongodb-org

**Comment by tteck** [comment]
🛑 Failure to comply with the following guideline.
> - If the default Linux distribution is not adhered to, script support will be discontinued.

```
    ____       __  _     
   / __ \___  / /_(_)___ 
  / /_/ / _ \/ __/ / __ \
 / ____/  __/ /_/ / /_/ /
/_/    \___/\__/_/\____/ 

Using Advanced Settings
Using Distribution: ubuntu
Using ubuntu Version: 20.04
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: petio
Using Disk Size: 4
Allocated Cores: 2
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Petio LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.112.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ MongoDB 4.4 Installed
 ✓ Installed Petio
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Petio should be reachable by going to the following URL.
         http://192.168.86.250:7777 
```

**Reply by nfribeiro** [reply]
You're right, I was doing it on a debian lxc. With Ubuntu 20.04 the installation was successful.
Sorry for the inconvenience and thanks for the quick reply.
